src/components/MangaControl.tsx

```diff
@@ -136,8 +137,13 @@ const MangaControl: React.FunctionComponent<Props> = (props: Props) => {
             <div className="title">Reading Mode</div>
           </div>
           <select className="subtitle dropdown" onChange={changeReaderMode} value={readerSettings.readerMode}>
-            {Object.values(ReaderMode).map((readerMode: string) =>
-              (<option className="subtitle" value={readerMode} key={readerMode}>{readerMode}</option>)
+            {Object.values(ReaderMode).map((readerMode: string) => {
+              if (mangaType === 'manga') {

```

Comment:
what is the purpose of this if

Reply:
looks like you're returning the same thing in both cases, merge them using a || or sm